### DISCUSSION TITLE
Making the Main Menu Icon be a separate plugin?

### DISCUSSION THREAD

**Discussion by SaneGhoul** [jessebofill/DeckWebBrowser#19] [Ideas]
I got some good use out of this plugin, but I kept finding myself opening up a separately installed browser so I could have a full desktop web browser with customized button bindings, extensions, password manager, and so on. But I dislike having my browser take up a spot on the home screen between my actual games and having it be in my main menu.

What if this plugin had a feature where you could have it open a separately installed browser like Chrome or Firefox instead? Perhaps it could be its own plugin where you could add any app/game to the Main Menu with a custom icon? Having a browser, Discord, a game wiki, or some other app be in the main menu but in an aesthetically consistent way that lets it avoid being on the home screen would be really nice.

**Comment by jessebofill** [comment]
I understand the drawbacks and desire for a full browser. It do think it would be a better idea to implement as a separate plugin but I myself am not interested in making it. If you’re not already in the decky discord you should check it out. You can make plugin suggestions there. Maybe someone else would be interested in it.
